web/src/hosta/Object.coffee

```diff
@@ -6,15 +6,16 @@ import Websocket from '../websocket/Websocket.coffee'
 export default class Object extends Component
   constructor:(props) ->
     super props
-    @name = @constructor.name
+    @object_props = {}
```

Comment:
This is props cache. If we got partial updates from data, they will be extended using this cache.